packages/webpack/template-webpack-plugin/test/cases/runtime-module/cache-events/index.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="vitest/globals" />
+
+import { add } from './lib-common.js';
+
+it('should have `__webpack_require__.__cache_events__`', () => {
+  expect(add(1, 2)).toBe(3);
+  expect(__webpack_require__['__cache_events__']).toBeTruthy();
```

Comment:
Can we also add a test to see if `__webpack_require__['__cache_events__']` does not exist when no chunk split is set.